conversion.go

```diff
@@ -0,0 +1,32 @@
+package seccomp // import "github.com/seccomp/containers-golang"
+
+import "fmt"
+
+var goArchToSeccompArchMap = map[string]Arch{
+	"386":         ArchX32,
+	"amd64":       ArchX86_64,
+	"amd64p32":    ArchX86,
+	"arm":         ArchARM,
+	"arm64":       ArchAARCH64,
+	"ppc":         ArchPPC,
+	"ppc64":       ArchPPC64,
+	"ppc64le":     ArchPPC64LE,
```

Comment:
Nit: sorting the keys would make the map easier to read.

Reply:
Yep, sorted them alphabetically.